src/js/main.js

```diff
@@ -31,12 +31,13 @@ var testSuites = [];
 var testFilters = [];
 var currentTest;
 
+// Populate the device selection drop down menu after the gUM dialog closes.
 document.querySelector('gum-dialog').addEventListener('closed', function() {
-  if (typeof MediaStreamTrack.getSources === 'undefined') {
-    console.log('getSources is not supported, device selection not possible.');
-  } else {
-    MediaStreamTrack.getSources(gotSources);
-  }
+  navigator.mediaDevices.enumerateDevices()
+  .then(gotSources)
```

Comment:
not clear to me if .then(... lines should be indented as expected in C++/java. (i.e. 4 spaces).
Anyhow if we have no linting errors to enforce any style preference I am fine whatever you pick.